spec/classes/postfix_spec.rb

```diff
@@ -334,12 +334,11 @@
             end
           end
           context 'when mta is enabled' do
-            let(:params) { { mta: true, mydestination: '1.2.3.4', relayhost: '2.3.4.5' } }
```

Comment:
Why remove?

Also, I think testing should be added to verify the intended behavior.

Reply:
Whith this change, $relayhost become like $satellite. So as you can't have $satellite and $mta define in the same time, you can't have $relayhost and $mta in same time. 
When you use the mta profile whit the relayhost, there is two correct way to do it : 
 - being call by satellite profile
 - being call directly
So this change is necessary to made test working.

Reply:
Since these items are mutually exclusive, there should be a test for them that shows the module fails as expected if they are both set.